src/main/java/frc/JoystickBindings.java

```diff
@@ -0,0 +1,31 @@
+package frc;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.button.Trigger;
+import frc.robot.Robot;
+import frc.joysticks.SmartJoystick;
+import frc.utils.utilcommands.InitExecuteCommand;
+
+public class JoystickBindings {
+
+	public enum JoystickBindSet {
```

Comment:
```suggestion
	public enum BindSet {
```
"joystick" is in the class name